03_analysis/compare_ffs.py

```diff
@@ -29,6 +29,7 @@
 import rdkit.Chem as Chem
 from rdkit.Chem import TorsionFingerprints
 import reader
+from collections import OrderedDict
```

Comment:
Was this import necessary?

Reply:
Oh, this was for `in_dict`. That dictionary is passed in from the `reader.py` module, which imports the `collections` library there. So I think this can be removed.